packages/components/button/src/Button/README.mdx

```diff
@@ -1,15 +1,14 @@
 ---
 title: 'Button'
+description: 'Start actions, such as “Add”, “Cancel” or “Save”.'
```

Comment:
Could you please elaborate on why we decided to change the long description (that we had in the content) with this one? Do we plan to do the same for each component page?

Also noticed that if the description is more than one line, it becomes hard to read
![image](https://user-images.githubusercontent.com/10744462/148969988-eb88d603-ed83-487a-900b-272c42d7fb26.png)

Reply:
>Also noticed that if the description is more than one line, it becomes hard to read

Oh! Good catch 
I used `Text` and didn't put any lineHeight there, I will fix it thanks!

>Could you please elaborate on why we decided to change the long description (that we had in the content) with this one? Do we plan to do the same for each component page?

@domarku designed this new page header that I think he didn't have time to show to the team yet
but the idea is to give short descriptions of what is the use of the component
I did the change just to in Button because I wanted to test the CSS and forgot to revert it before opening the PR, my bad
I will revert it and we can implement it later

and yes, the idea is that all components would follow this pattern

again, let me revert this change in Button.mdx and if we go with this we can simply add it later

Thanks for catching these leftovers and bugs 🙌